### PR TITLE
send message

### PR DESCRIPTION
**Pull request by pythexcel**

### CODE REVIEW

File: app/api/message_create.py [hunk 1]
```diff
@@ -29,12 +28,13 @@ def notification_message():
...
                 "message_key": MSG_KEY,
-                "message_origin": MSG_ORIGIN,
+                "message_origin": message_origin,
                 "message_type": MSG_TYPE,
                 "working":Working,
                 "message_color": MSG_Color,
                 "slack_channel":slack_channel,
-                "email_group":email_group
+                "email_group":email_group,
+                "channel":channel
```
Comment (xmagestore) [inline app/api/message_create.py]: this should be channels not channel

File: app/api/notify.py [hunk 2]
```diff
@@ -17,11 +17,22 @@ def dispatch():
...
+    print(missed_req.items())
+    print(message_detail)      
+    # below will checki if message detail is completely empty return data from json or else if its any value is none replace it from json data
+    if message_detail is not None:
+        update = message_detail.update((k,v) for k,v in missed_req.items() if v is not None)
+    else:
+        message_detail = missed_req
     if message_detail and message_detail['message_type'] is not None:   
             message = message_detail['message']
+            slack_channels = message_detail['slack_channel']
```
Comment (xmagestore) [inline app/api/notify.py]: remove this from here. at this stage we don't even know if message is to be delivered via slack. why are u accessing slack channel here

File: app/api/message_create.py (hunk 1 above)
Comment (pythexcel) [reply app/api/message_create.py]: ok sir will change to channels

File: app/api/notify.py (hunk 2 above)
Comment (pythexcel) [reply app/api/notify.py]: ok will remove this

**Pull request merged**